internal/presenters/components.go

```diff
@@ -47,19 +49,45 @@ func RenderError(err snyk_errors.Error) string {
 
 	if err.StatusCode > 0 {
 		body = append(body, lipgloss.JoinHorizontal(lipgloss.Top,
-			label.Render("HTTP:"),
-			value.Render(strconv.Itoa(err.StatusCode)),
+			label.Render("Status:"),
+			value.Render(strconv.Itoa(err.StatusCode)+" "+http.StatusText(err.StatusCode)),
 		))
 	}
 
+	if len(err.Description) > 0 {
+		desc := err.Description
+		re := regexp.MustCompile("\n+")
+		lines := re.Split(desc, -1)
+
+		if len(lines) > 1 {
+			lines = lines[0:2]
+			for i, l := range lines {
+				lines[i] = strings.Trim(l, " \n")
+			}
+			desc = strings.Join(lines, " ")
+		}
+
+		body = append(body, lipgloss.JoinHorizontal(lipgloss.Top,
+			label.Render("Details:"),
+			value.Copy().Width(80).Render(desc),
+		))
+	}
+
+	title := strings.TrimSpace(err.Title)
+	if len(err.ErrorCode) > 0 {
+		link := "https://docs.snyk.io/scan-with-snyk/error-catalog#" + strings.ToLower(err.ErrorCode)
```

Comment:
Suggestion: let's move the url to a constant, it might not change but it might be good to have it re-usable.

Reply:
maybe even two constants, one for the docs base URL and one for the path